docs/faq.rst

```diff
@@ -49,3 +49,27 @@ There are two ways to configure Ansible to keep a logfile:
   :command:`ansible-playbook` will be invoked.
 
 For more information, see :ref:`ansible:DEFAULT_LOG_PATH`.
+
+
+.. _ansible-user-root-detection:
+
+Can I use the 'root' user to deploy MetalK8s to servers?
+--------------------------------------------------------
+During deployment of MetalK8s, a set of tasks are executed to bring the target
+system in line with the `RHEL7 STIG`_ security guidelines, using the
+`ansible-hardening`_ role. STIG rule `V-72247`_ disallows remote SSH access
+using the `root` user. As such, if MetalK8s were deployed using `root` to
+access a remote system, this would effectively disable access to said server.
+
+We integrated a check in the playbook to assert `ansible_user` is not set to
+`root` on any of the target hosts, so deployment will abort early if this
```

Comment:
to abort any deployment if this configuration is detected.

Reply:
Done,  but replaced `any` with `the` as discussed offline.